Packs/PrismaCloud/ReleaseNotes/2_0_10.md

```diff
@@ -0,0 +1,13 @@
+#### Classifiers
+- **Prisma Cloud - Classifier**
+- Added a new classification rule for the prisma policy: "AWS Security Group allows all traffic on SSH port (22)" with ID: "617b9138-584b-4e8e-ad15-7fbabafbed1a"
+#### Mappers
+##### Prisma Cloud - Incoming Mapper
+- Fixed the "Account Name" mapping under the "AWS EC2 Instance Misconfiguration" incident type
+
+#### Playbooks
+##### Prisma Cloud Remediation - AWS EC2 Instance Misconfiguration
+- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts
+
+##### Prisma Cloud Remediation - AWS EC2 Security Group Misconfiguration
+- Updated the playbook to investigate the "AWS Security Group allows all traffic on SSH port (22)" alerts
```

Comment:
We have a new validation for the release notes that checks that every section starts with one of the following prefixes: 

```RN_TEMPLATES = {
    # prefixes
    'Added support for',
    'Added the',
    'Fixed an issue',
    'Improved implementation',
    'Updated the Docker image to',
    'You can now',
    'Deprecated. ',
    'Deprecated the ',

    # full line
    'Documentation and metadata improvements.',
}

BANNED_TEMPLATES = {
    'maintenance and stability enhancements.',
    'stability and maintenance enhancements.',
}
```


Can you please edit your release notes in accordance with this new validation?